crates/anvil/src/lib.rs

```diff
@@ -246,6 +246,9 @@ impl NodeHandle {
                     .join(", ")
             )
         }
+        if let Some(ipc_path) = self.ipc_path() {
```

Comment:
looks good! let's move this into the `if !self.config.silent` check above so it respects it